man/preupgrade-assistant-api.1

```diff
@@ -4,119 +4,119 @@
 .\"
 .TH preupgrade-assistant-api 1 2016-09-01 "" "Linux User's Manual"
 .SH NAME
-preupgrade-assistant-api \- Preupgrade assistant API provides a set of functions
-and variables which can be used for contents writing.
+preupgrade-assistant-api \- Preupgrade Assistant API provides a set of functions
+and variables which can be used for creating modules.
 
 .SH AVAILABLE FUNCTIONS
 API provides a set of functions which can be used for better handling of your data.
 
-\fBcheck_applies_to\fP - Function can be used for specific RPM package detection. If the RPM package does not exists then check script exit with \fBexit_not_applicable\fP return code.
+\fBcheck_applies_to\fP - The function can be used for specific RPM package detection. If the RPM package does not exist, check the script exit with the \fBexit_not_applicable\fP return code.
 
-\fBcheck_root\fP - Function checks whether user is root.
+\fBcheck_root\fP - The function checks whether the user is root.
 
-\fBsolution_file\fP - Function adds a message to solution file defined in INI file.
+\fBsolution_file\fP - The function adds a message to the solution file defined in the INI file.
 
-\fBbackup_config_file\fP - Function backups a config file to \fB/root/preupgrade\fP directory.
+\fBbackup_config_file\fP - The function backs up a config file to the \fB/root/preupgrade\fP directory.
```

Comment:
Which word? I can't see any suspicious word there (can't say that about the old text).